app/services/orphan_s3_derivatives.rb

```diff
@@ -121,9 +145,38 @@ def parse_s3_path(s3_path)
   end
 
 
-  def orphaned?(asset_id, derivative_key, shrine_id)
-    return true unless asset_id.present? && derivative_key.present?
+  # Combined audio derivatives are stored at an s3_path of:
```

Comment:
This comment is confusing. What I meant is that s3 paths for c.a.d's start in an invariable string `combined_audio_derivatives`.